modules/virt-creating-linux-bridge-nncp.adoc

```diff
@@ -3,15 +3,15 @@
 // * virt/virtual_machines/vm_networking/virt-attaching-vm-multiple-networks.adoc
 
 :_content-type: PROCEDURE
-[id="virt-configuring-linux-bridge-nncp_{context}"]
-= Creating a Linux bridge using a node network configuration policy
+[id="virt-creating-linux-bridge-nncp_{context}"]
 
-As a network administrator, you can create a Linux bridge interface on nodes in the cluster by applying a `NodeNetworkConfigurationPolicy` manifest to the cluster.
+= Creating a Linux bridge node network configuration policy
+
+Use a `NodeNetworkConfigurationPolicy` manifest YAML file to create the Linux bridge.
 
 .Procedure
 
-. Create the `NodeNetworkConfigurationPolicy` manifest. This YAML file is an example of a manifest for a Linux bridge interface.
-It includes samples values that you must replace with your own information.
+. Create the `NodeNetworkConfigurationPolicy` manifest. This example includes sample values that you must replace with your own information.
 
 [source,yaml]
 ----
```

Comment:
This is out of scope but I think we can consider this comment as fixing typos: Change "ipv4" to "IPv4" and "stp" to "STP".

"The node NIC to which the bridge attaches." This is like "displays", which we are not supposed to use as an intransitive verb. Change to "The node NIC to which the bridge is attached" or something similar.

Reply:
Fixed the cap issues in the callouts. Didn't want to change the YAML obviously. Good catch on the int. verb.